CONTRIBUTING.md

```diff
@@ -59,6 +59,15 @@ If you want to work on github.com/01org/ciao and your fork on the same workstati
 
 Once you've finished making your changes push them to your fork and send the PR via the github UI.  If you don't need to maintain the github.com/01org/ciao repo and your fork on the same workstation you can skip steps 2 and 3.
 
+## Quality Controls
+
+We request you give quality assurance some consideration by:
+* Adding go unit tests for changes where it makes sense.
+* Using our [test-cases](https://github.com/01org/ciao/tree/master/test-cases) package to run unit tests because a simple ```gotest ./...``` can result in multiple packages' unit tests running in parallel which may be unsafe.  Simply replace ```go test``` with ```test-cases``` in your workflow.
```

Comment:
I think ciao-controller and testutils and ciao-scheduler can't 'go test' reliably in parallel, and possibly the networking as well.